book/09-git-and-other-scms/sections/client-svn.asc

```diff
@@ -299,18 +299,18 @@ First, rewinding head to replay your work on top of it...
 Fast-forwarded master to refs/remotes/origin/trunk.
 ----
 
-Running `git svn rebase` every once in a while makes sure your code is always up to date.
-You need to be sure your working directory is clean when you run this, though.
-If you have local changes, you must either stash your work or temporarily commit it before running `git svn rebase` – otherwise, the command will stop if it sees that the rebase will result in a merge conflict.
+Ang pagpapatakbo ng `git svn rebase` sa bawat pagkakataon ay sinisiguradong ang iyong code ay palaging napapanahon.
+Kailangan mong siguraduhin na ang iyong tinatrabahong direktoryo ay malinis kapag ikaw ay nagpatakbo nito, bagaman.
+Kung mayroon kang lokal na mga pagbabago, kailangan mong alinmang i-stash ang iyong trabaho o pansamantalang i-commit ito bago patakbuhin `git svn rebase` – kung hindi man, ang utos ay hihinto kung nakikita nito na ang rebase ay magreresulta ng isang merge na salungatan.
 
-===== Git Branching Issues
+===== Mga Isyu sa Pag-branch ng Git
 
-When you've become comfortable with a Git workflow, you'll likely create topic branches, do work on them, and then merge them in.
-If you're pushing to a Subversion server via `git svn`, you may want to rebase your work onto a single branch each time instead of merging branches together.
-The reason to prefer rebasing is that Subversion has a linear history and doesn't deal with merges like Git does, so `git svn` follows only the first parent when converting the snapshots into Subversion commits.
+Kapag ikaw ay naging komportable na sa isang Git na daloy ng trabaho, malamang ikaw ay makakalikha ng paksa na mga branch, gumawa ng trabaho sa mga ito, at pagkatapos ay papasok na i-merge ang mga ito.
+Kung ikaw ay nag-push sa isang Subversion na server sa pamamagitan ng `git svn`, malamang gusto mong i-rebase ang iyong trabaho sa isang solong branch sa bawat oras sa halip na sama-samang i-merge ang mga branch.
```

Comment:
I'm not sure with this but do we need to translate "workflow" to "daloy ng trabaho"? If we should, I suggest to change this "komportable na sa isang Git na daloy ng trabaho" to "komportable na daloy ng trabaho sa isang Git". It just sounds better I guess.... and we could eliminate the "na - na"...

Reply:
I used "daloy ng trabaho" because I think it suggests the equivalent meaning "workflow". But, let me check the conventions used by other contributions so I could follow what others are doing. Thank you btw, nice suggestion.

Reply:
Hey @cherryloua . Based on my observation, everyone here is retaining "workflow". Therefore, I'll be changing my translation based on your suggestion. Thank you for pointing that one out.